lib/scope.js

```diff
@@ -1554,11 +1558,9 @@ module.exports = {
     let scenario = [ scope.scenario_id, current_report_obj ];  // Mimic how loop works with `Map`
     let report = await scope.getPrintableScenario( scope, { scenario });
     let all_are_included = true;
-    for ( let content of expected ) {
-      if ( !content.includes( expected )) {
-        all_are_included = false;
-      }
-      expect( report ).to.contain( content );
+    for ( let one_expectation of expected ) {
+      if ( !report.includes( one_expectation )) { all_are_included = false; }
+      expect( report ).to.contain( one_expectation );
```

Comment:
Not super important, but this seems slightly inefficient. Like 1562 will loop through all items in report once, and then 1563 will do so again. Moving this `expect` statement inside the above `if` block will make this slightly more efficient, because then the second loop won't happen every time.